common/scala/src/main/scala/org/apache/openwhisk/core/connector/Message.scala

```diff
@@ -338,7 +339,8 @@ object Activation extends DefaultJsonProtocol {
       "kind",
       "conductor",
       "memory",
-      "causedBy")
+      "causedBy",
+      "size")
```

Comment:
> Enables including the response size in ActivationResponse, WhiskActivation and eventually in Activation object sent to events topic

Why not not just calculate this where needed and avoid changing the message or the interfaces below?

Reply:
That would require serializing the response while constructing the Activation object to send the User event.

To avoid that overhead I had to pass the size all the way.